article/app/views/fragments/articleAsideSlot.scala.html

```diff
@@ -0,0 +1,17 @@
+@(
+    shouldShowAds: Boolean,
+    optionalSlotSizes: Seq[String],
```

Comment:
With the simplification of the `optionalAsideSizes` method, you can get rid of that parameter